### ISSUE TITLE
Radio button value not updating correctly

### ISSUE THREAD

**Issue by oliverfaessler** [TomSchimansky/CustomTkinter#75]
### Expected behavior
In the callback of the radio button the variable should update with the fisrt click on the radio button.

### Actual behavior
The variable contains the value of the previous cklick. The radio butten has to be clicked twice to update the variable correctly.

### Example
``` python
import tkinter
import customtkinter  # <- import the CustomTkinter module

customtkinter.set_appearance_mode("dark")  # Modes: "System" (standard), "Dark", "Light"
customtkinter.set_default_color_theme("blue")  # Themes: "blue" (standard), "green", "dark-blue"

root_tk = customtkinter.CTk()  # create CTk window like you do with the Tk window (you can also use normal tkinter.Tk window)
root_tk.geometry("400x200")
root_tk.title("CustomTkinter Test")


def radiobutton_function():
    print("radiobutton:", radiobutton_var.get())


y_padding = 13

radiobutton_var = tkinter.IntVar(value=1)

radiobutton_1 = customtkinter.CTkRadioButton(master=root_tk, variable=radiobutton_var, value=1,command=radiobutton_function)
radiobutton_1.pack(pady=y_padding, padx=10)

radiobutton_2 = customtkinter.CTkRadioButton(master=root_tk, variable=radiobutton_var, value=2,command=radiobutton_function)
radiobutton_2.pack(pady=y_padding, padx=10)

root_tk.mainloop()
```

**Comment by TomSchimansky** [comment]
This is indeed a bug, thanks for reporting this, I will fix this with the next version.

**Comment by felipetesc** [comment]
I don't know if this is a bug. I changed the code and by adding a print before the radio click it printed correctly the var value:
```python 
import tkinter
import customtkinter  # <- import the CustomTkinter module

customtkinter.set_appearance_mode("dark")  # Modes: "System" (standard), "Dark", "Light"
customtkinter.set_default_color_theme("blue")  # Themes: "blue" (standard), "green", "dark-blue"

root_tk = customtkinter.CTk()  # create CTk window like you do with the Tk window (you can also use normal tkinter.Tk window)
root_tk.geometry("400x200")
root_tk.title("CustomTkinter Test")
def radiobutton_function():
    print("radiobutton:", radiobutton_var.get())
y_padding = 13
radiobutton_var = tkinter.IntVar(value=1)
radiobutton_var2 = tkinter.IntVar()
radiobutton_1 = customtkinter.CTkRadioButton(master=root_tk, variable=radiobutton_var, value=1,command=radiobutton_function)
radiobutton_1.pack(pady=y_padding, padx=10)
radiobutton_2 = customtkinter.CTkRadioButton(master=root_tk, variable=radiobutton_var, value=2,command=radiobutton_function)
radiobutton_2.pack(pady=y_padding, padx=10)
################################
print("radiobutton:", radiobutton_var.get())
################################
root_tk.mainloop()
```

**Comment by felipetesc** [comment]
Maybe this issue can be closed.

**Comment by TomSchimansky** [comment]
It was a bug, I fixed it, and when the next version gets uploaded to pip, I will close this issue.

**Comment by TomSchimansky** [comment]
It's now fixed with version 4.0.0
